### ISSUE TITLE
dotnet-ef 2.1.4 is targeting runtime 2.1.5

### ISSUE THREAD

**Issue by wli3** [dotnet/efcore#13544]
https://github.com/dotnet/cli/issues/10112

If the `dotnet-ef` does not intent to align with runtime version please close this issue.

Describe what is not working as expected.

### Steps to reproduce
Download dotnet-ef 2.1.4 nupkg, https://www.nuget.org/api/v2/package/dotnet-ef/2.1.4

find the file `dotnet-ef.2.1.4\tools\netcoreapp2.1\any\dotnet-ef.runtimeconfig.json`

it is targeting
```json
"framework": {
      "name": "Microsoft.NETCore.App",
      "version": "2.1.5"
    }
```

### Further technical details
dotnet-ef 2.1.4

**Comment by ajcvickers** [comment]
@wli3 This is by-design. Patch versions were only incremented when a patch was release. ASP.NET Core 2.1.x has  been patched one more time than EF Core 2.1.x, and hence ASP.NET Core is at 2.1.5 and EF Core is at 2.1.4.

I managed to get this changed for future releases, so if ASP.NET Core and EF Core are patched in the same release, then going forward they will have the same version.

**Comment by alanrezendee** [comment]
Understood! 
Thank you guys!
